src/array-methods/4/task.js

```diff
@@ -0,0 +1,21 @@
+const trips = [
+    { location: "Rome", days: 7, cost: 1400 },
+    { location: "Sydney", days: 14, cost: 2800 },
+    { location: "Maui", days: 10, cost: 2500 }
+];
+
+trips.forEach(trip => {
+    trip.costPerDay = trip.cost / trip.days;
+})
```

Comment:
`forEach` isn't wrong here. However: how would you/would it be possible at all to do it with `map`?